src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/BuildAssetsUtils.cs

```diff
@@ -546,13 +603,22 @@ public static string GetMSBuildFilePathForPackageReferenceStyleProject(PackageSp
                     buildCrossTargetsGroup.RootName = MSBuildRestoreItemGroup.ImportGroup;
                     buildCrossTargetsGroup.Position = 0;
 
-                    buildCrossTargetsGroup.Items.AddRange(sortedPackages.SelectMany(pkg =>
-                        pkg.Key.BuildMultiTargeting.WithExtension(TargetsExtension)
-                            .Where(e => pkg.Value.Exists())
-                            .Select(e => pkg.Value.GetAbsolutePath(e)))
-                            .Where(path => multiTargetingImportsAdded.Add(path))
-                            .Select(path => GetPathWithMacros(path, repositoryRoot))
-                            .Select(GenerateImport));
+                    foreach (var pkg in sortedPackages)
```

Comment:
Is this the same implementation as the previous ones?

Reply:
Appears like it to me...